src/styles/problem.scss

```diff
@@ -111,3 +111,34 @@ td.problem-cell {
 .problem-div-4 {
     background: grey;
 }
+.div-heading-container {
+    position: absolute;
+    top: -12px;
+    right: 15px;
+    padding: 8px;
+    border-radius: 4px;
+    width: 300px;
+  }
```

Comment:
Add spaces here is not correct.

Reply:
Let's check all unfolded comments, and click "Resolve conversation" if you think it has been fixed.